pkgs/development/libraries/libpsl/default.nix

```diff
@@ -61,6 +62,7 @@ stdenv.mkDerivation rec {
     "--with-psl-distfile=${publicsuffix-list}/share/publicsuffix/public_suffix_list.dat"
     "--with-psl-file=${publicsuffix-list}/share/publicsuffix/public_suffix_list.dat"
     "--with-psl-testfile=${publicsuffix-list}/share/publicsuffix/test_psl.txt"
+    "PYTHON=${lib.getDev buildPackages.python3}/bin/python3"
```

Comment:
Why getDev,  dev output usually doesn't have bin

Reply:
Oh dear, now THAT is a typo. I have no idea how this worked in my testing...